cmake/exiv2Config.cmake.in

```diff
@@ -0,0 +1,10 @@
+@PACKAGE_INIT@
+
+cmake_minimum_required(VERSION 3.16.3)
+include(CMakeFindDependencyMacro)
+
+find_dependency(ZLIB REQUIRED)
```

Comment:
This is the find dependency call I mentioned that was missing. This should only be added to the generated file if it was built with PNG support. 

Here is one approach: 
https://discourse.cmake.org/t/installing-exporting-projects-that-conditionally-use-optional-dependencies/5194/2

Reply:
I used this approach. With `EXIV2_ENABLE_PNG` turned on, the following config file gets generated:
```
cmake_minimum_required(VERSION 3.16.3)
include(CMakeFindDependencyMacro)

if(ON) # if(EXIV2_ENABLE_PNG)
  find_dependency(ZLIB REQUIRED)
endif()

include("${CMAKE_CURRENT_LIST_DIR}/exiv2Export.cmake")

check_required_components(exiv2)
```